### PR TITLE
Issue 1199 - Fast objects (Archive) unit test (manual)

### PR DESCRIPTION
**Pull request by lstochlak**
Fast objects (Archive) unit test (manual)

### CODE REVIEW

**Comment by CesarCoelho** [comment]
Hi,

What I think it is missing from your Unit Test is to test that the "Fast" objects can be used concurrently.
This means you will need to spawn a few hundred threads and make calls on it all at the same time.

Best regards,
César Coelho

File: core/mo-services-impl/ccsds-com-impl/src/test/java/esa/mo/com/impl/provider/ArchiveManagerTest.java [hunk 1]
```diff
@@ -0,0 +1,199 @@
+//------------------------------------------------------------------------------
```
Comment (dmarszk) [inline core/mo-services-impl/ccsds-com-impl/src/test/java/esa/mo/com/impl/provider/ArchiveManagerTest.java:1]: Ensure proper license header is followed

**Comment by dmarszk** [comment]
> Hi,
> 
> What I think it is missing from your Unit Test is to test that the "Fast" objects can be used concurrently. This means you will need to spawn a few hundred threads and make calls on it all at the same time.
> 
> Best regards, César Coelho

Indeed extensive Unit Test would include that, but the goal here was not to increase coverage, but to benchmark the performance - see the ticket https://gitlab.com/esa/NMF/nmf-issues/-/issues/1199

**Review by dmarszk — CHANGES_REQUESTED** [review]
Needs a license header update

File: core/mo-services-impl/ccsds-com-impl/src/test/java/esa/mo/com/impl/provider/ArchiveManagerTest.java (hunk 1 above)
Comment (lstochlak) [reply core/mo-services-impl/ccsds-com-impl/src/test/java/esa/mo/com/impl/provider/ArchiveManagerTest.java]: Done

**Pull request merged**